ledger-service/http-json-cli/src/main/scala/com/daml/http/OptionParser.scala

```diff
@@ -165,4 +170,16 @@ class OptionParser(getEnvVar: String => Option[String], supportedJdbcDriverNames
     .optional()
     .valueName(WebsocketConfig.usage)
     .text(s"Optional websocket configuration string. ${WebsocketConfig.help}")
+
+  // TODO: Refactor out as this is duplicated from the sandbox code
+  opt[String]("log-level")
+    .optional()
+    .validate(l =>
+      Either
+        .cond(KnownLogLevels.contains(l.toUpperCase), (), s"Unrecognized logging level $l")
+    )
+    .action((level, c) => c.copy(logLevel = Some(LogLevel.toLevel(level.toUpperCase))))
+    .text(
+      "Default logging level to use. Available values are INFO, TRACE, DEBUG, WARN, and ERROR. Defaults to INFO."
```

Comment:
```suggestion
      s"Default logging level to use. Available values are ${knownLogLevels.mkString(", ")}. Defaults to INFO."
```

Reply:
I will address this with the refactoring PR.